ansible/roles/zabbix_es/templates/ESzabbix.py.j2

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
```

Comment:
That one is not from ansible galaxy so I not in the same way. I was hoping to add a requirements file and a line to the `init-ansible` command that could auto populate the `roles_path` if we ever changed control machines or needed to set up an new one and that would not work with the es role. We could just manually move it if you feel that is cleaner